src/prop_tests/dotNotation_test.js

```diff
@@ -0,0 +1,31 @@
+var QUnit = require("steal-qunit");
+
+var set = require('../set-core'),
+  props = require("../props");
+
+QUnit.module("can-set props.boolean");
+
+/*
+ * For the dotNotation prop, we define sets like so:
+ *
+ * For a property 'n.p', with value 'IL'
+ * x ∈ X | x.n.p = 'IL'
+ *
+ */
+test('dotNotation set membership', function() {
+  /*
+   * For a property 'n.p', with value 'IL'
+   * x ∈ X | x.n.p == 'IL'
+   */
+  var prop = props.dotNotation('n.p'),
+      alg = new set.Algebra(prop),
+      res = alg.has({'n.p': 'IL'}, {n:{p:'IL'}});
+  ok(res, "object with nested property is member of set using dotNotation");
```

Comment:
Was also looking for a way to test the unpredictable argument order handling. Tried an alg.has with the arguments reversed, and my comparator returns true in that case, but alg.has was still returning false. I'm guessing another 'default' prop was doing its own check and returning false.

Anyone have an idea how I could test that via an algebra operation?